src/Switcher.java

```diff
@@ -0,0 +1,15 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Switcher {
+
+    private Map<GameId, GameStrategy> gameList = new HashMap<>();
+
```

Comment:
you can define your game-types here, and call this class like Context or ContextHolder